### PR TITLE
reworking container guides

### PR DESCRIPTION
**Pull request by ChristinaLK**
Split out making a container from using a container in OSG. 

Links added, needs polish. 

Please let me know if it flows smoothly or if anything is missing.

### CODE REVIEW

**Comment by ChristinaLK** [comment]
@rynge @CarrieBrown

**Comment by rynge** [comment]
Looks great!

File: start/software/container-how-to.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+The first line of this file should include the keyword `FROM` and then 
+the name of a Docker image (from Docker Hub) you want 
+to use as your starting point. If using the OSG's Ubuntu Xenial image that 
+would look like this: 
+
+	FROM opensciencegrid/osgvo-ubuntu-xenial
+
+Then, for each command you want to run to add libraries or software, use the 
+keyword `RUN` and then the command. Sometimes it makes sense to string
```
Comment (CarrieBrown) [inline start/software/container-how-to.md]: When does it make sense to string together commands vs using multiple RUN commands? Perhaps a brief sentence stating the use cases would be helpful.

File: start/software/container-how-to.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+Once you have installed all the software, you simply `exit`
+
+    [root@740b9db736a1 /]# exit
+
+Now you can commit the changes to the image: 
+
+    docker commit <docker_session_name_here>
+
+Alternatively, you can use the sessions hash (`740b9db736a1` in the above example).
```
Comment (CarrieBrown) [inline start/software/container-how-to.md]: Perhaps: "you can use the sessions hash as found in the command prompt (`740b9db736a1` in the above example)."

File: start/resources/singularity-containers.md [hunk 3]
```diff
@@ -88,170 +27,153 @@ tools and libraries. The images are are:
...
-
-#### Editing the `Dockerfile`
+our own distribution system (see below). 
 
-To create an image locally using a `Dockerfile`, 
-first download the `Dockerfile` for your desired flavor from 
-[OSG GitHub repository](https://github.com/opensciencegrid), 
-see table above. For example, if you want to base the image on our Ubuntu
-Xenial image, 
+See [this page][container-howto] for how to create a Docker image on your own computer and
```
Comment (CarrieBrown) [inline start/resources/singularity-containers.md]: This should be `[this page](container-how-to)`

File: start/resources/singularity-containers.md [hunk 4]
```diff
@@ -88,170 +27,153 @@ tools and libraries. The images are are:
...
+[cvmfs-singularity-sync repository](https://github.com/opensciencegrid/cvmfs-singularity-sync), 
+or contact
+[support@osgconnect.net](mailto:support@osgconnect.net)
+and we can help you.
 
-    $ docker push namespace/repository_name
+Once your submission has been accepted, it will be automatically converted to a Singularity 
+image and pushed to the CVMFS Singularity repository.  Note: some 
+common Dockerfile features, like ENV and ENTRYPOINT, are ignored when the Docker 
+image is converted to a Singularity image. See our [how to guide]() for more details
```
Comment (CarrieBrown) [inline start/resources/singularity-containers.md]: Are we planning to build the 'how to guide' or should we just direct them to the "Special Cases" section for details here?

File: start/resources/singularity-containers.md [hunk 5]
```diff
@@ -88,170 +27,153 @@ tools and libraries. The images are are:
...
-please run
 
-    [root@740b9db736a1 /]# mkdir -p /cvmfs 
+## Frequently Asked Questions / Common Issues
 
-Once you have installed all the software, you simply `exit`
+### I already have a Singularity container, not a Docker one
 
-    [root@740b9db736a1 /]# exit
+?
```
Comment (CarrieBrown) [inline start/resources/singularity-containers.md]: This needs to be elaborated on. Unfortunately, I don't know the answer to this myself :)

File: start/resources/singularity-containers.md (hunk 3 above)
Comment (ChristinaLK) [reply start/resources/singularity-containers.md]: I'm using reference style links.

File: start/resources/singularity-containers.md (hunk 4 above)
Comment (ChristinaLK) [reply start/resources/singularity-containers.md]: This is meant to be the container-how to -- I'll make it clear it's under special cases.

File: start/resources/singularity-containers.md (hunk 5 above)
Comment (ChristinaLK) [reply start/resources/singularity-containers.md]: my thoughts exactly. 😂

File: start/resources/singularity-containers.md [hunk 6]
```diff
@@ -92,7 +92,8 @@ and we can help you.
 Once your submission has been accepted, it will be automatically converted to a Singularity 
 image and pushed to the CVMFS Singularity repository.  Note: some 
 common Dockerfile features, like ENV and ENTRYPOINT, are ignored when the Docker 
-image is converted to a Singularity image. See our [how to guide]() for more details 
+image is converted to a Singularity image. See our the "Special Cases" section of our 
+[how to guide][container-howto] for more details
```
Comment (CarrieBrown) [inline start/resources/singularity-containers.md]: I think you need another hyphen here - `container-how-to`

**Pull request merged**